.env.example

```diff
@@ -1,7 +1,15 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-PORT=
-
-DB_TEST_NAME=barefoot_test
+DB_USERNAME=postgres
+DB_PASSWORD=db_password
+DB_NAME=barefoot_nomad
+DB_HOST=127.0.0.1
+PORT=""
+DB_TEST_NAME=barefoot_test
+TOKENEXPIRY=18000
+EMAIL=
+CLIENT_URL="https://firestar_backend.herokuapp.com"
+HOST="localhost"
+DATABASE_URL_TEST=""
+NODE_ENV="test"
+YOUR_EMAIL=""

```

Comment:
What is the purpose of the `YOUR_EMAIL`?

Reply:
this has been updated in the PR.

YOUR_EMAIL="" (description: Creates a user account with this email to test reset password link)
SOME_PASSWORD="" (description: Create user account with password, Must include more than 3 characters)

thanks.

Reply:
are these temporary ENVs?

Reply:
And also include the description in the `.env.example file` next to the variables. e.g `YOUR_EMAIL=description`

Reply:
The description is updated. Yes they are temporary, the normal workflow is for a user to sign up before it can make a POST to resetpassword.